src/Http/Headers/src/SetCookieHeaderValue.cs

```diff
@@ -200,7 +200,7 @@ public override string ToString()
 
         if (MaxAge.HasValue)
         {
-            maxAge = HeaderUtilities.FormatNonNegativeInt64((long)MaxAge.GetValueOrDefault().TotalSeconds);
+            maxAge = ((long)MaxAge.GetValueOrDefault().TotalSeconds).ToString(NumberFormatInfo.InvariantInfo);
```

Comment:
This allocates now for `0`. It would be better to update `HeaderUtilities.FormatNonNegativeInt64` to allow negative values.

Reply:
That's kindof ironic :-)

FormatNonNegativeInt64 is used in several places that shouldn't use negatives. We'd want a new method for the new behavior.

Reply:
Sounds good, so should I create a new method `HeaderUtilities.FormatInt64`? I don't want to break other validations that might depend on the current behavior of `HeaderUtilities.FormatNonNegativeInt64`

Reply:
Yes please